.java-version

```diff
@@ -0,0 +1 @@
+openjdk64-11.0.2
```

Comment:
❓ Where is this used?

Reply:
Pretty sure jenv uses this. We could add it to the .gitignore